index.html

```diff
@@ -1076,26 +1076,27 @@ <h2>Basic Concepts</h2>
         <h3>Contexts</h3>
 
         <p>
-When two software systems need to exchange data, they need to use terminology that both
-systems understand.
-As an analogy, consider how two people communicate. Both people must use the same
-language and the words they use must mean the same thing to each other. This may be
-referred to as "the context of a conversation".
+When two software systems need to exchange data, they need to use terminology
+that both systems understand. As an analogy, consider how two people
+communicate. Both people must use the same language and the words they use must
+mean the same thing to each other. This could be referred to as
```

Comment:
`could` -> `might` to avoid [RFC 2119](https://tools.ietf.org/html/rfc2119) interpretation issues

Reply:
Done.